src/probnum/diffeq/odesolution.py

```diff
@@ -71,7 +71,17 @@ class ODESolution(FiltSmoothPosterior):
     """
 
     def __init__(self, times, rvs, solver):
-        self._kalman_posterior = KalmanPosterior(times, rvs, solver.gfilt)
+
+        # try-except is a hotfix for now:
```

Comment:
yes, it is there to make the current implementation testable which I think is a useful hotfix. Thoughts? 

> What would the future look like roughly, do you want a general ODESolution base class, and a ODEFilterSolution subclass?

see my above comment, but in general I prefer the former. This will make things more robust (only a single set of tests is required) and will keep information about how to interpolate where it needs to be anyway (with the solver). Thoughts?

Reply:
I think the Euler example is really nice to keep in mind not to overfit to the ODE filters, so I think the *hackyness* of the hotfix is worth it :) 

I'm not yet 100% sure if the interpolation should really be part of the solver itself, but I'm also not completely against it. Let's just discuss the future of the ODESolution class in person someday :)